.gitignore

```diff
@@ -0,0 +1,3 @@
+#dependencies
+/node_modules
+text.md
```

Comment:
- [ ] This file has not yet been ignored by Github, probably because there is a typo in the file name. It should be `test.md` not `text.md`. ℹ️